examples/Example/src/prot_siblings/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import {
+  GestureMonitor,
+  useGesture,
+  Gesture,
+} from 'react-native-gesture-handler';
+import { useState } from 'react';
+
+function getState(s: number) {
```

Comment:
This function is unused.

Reply:
Removed in 75e03c1b7b8e9ca64a07060c549319e5d70dabfa